cmd/cloud-controller-manager/app/apis/config/v1alpha1/defaults.go

```diff
@@ -45,4 +46,6 @@ func SetDefaults_CloudControllerManagerConfiguration(obj *CloudControllerManager
 
 	// Use the default RecommendedDefaultGenericControllerManagerConfiguration options
 	kubectrlmgrconfigv1alpha1.RecommendedDefaultGenericControllerManagerConfiguration(&obj.Generic)
+	// Use the default SetDefaultsfuncServiceControllerConfiguration options
+	serviceconfigv1alpha1.SetDefaultsfuncServiceControllerConfiguration(&obj.ServiceController)
```

Comment:
this must be `RecommendedDefaultServiceControllerConfiguration`, please fix this for all funcs